codalab/worker/docker_utils.py

```diff
@@ -141,9 +141,11 @@ def start_bundle_container(
         # nvidia-docker runtime uses this env variable to allocate GPUs
         environment['NVIDIA_VISIBLE_DEVICES'] = ','.join(gpuset) if gpuset else 'all'
 
+    # Name the container same as the UUID to avoid network name clashes
     container = client.containers.run(
         image=docker_image,
         command=docker_command,
+        name=uuid,
```

Comment:
maybe prefix this with `run-`?

Reply:
Do we want to prefix this with `codalab` something?

Reply:
`codalab-run-`?

Reply:
I think we should use `codalab_run_` because we use underscores to name other containers (`codalab_frontend_1`, etc.)

Reply:
sounds good

Reply:
Done